SQF/dayz_code/system/zombie_agent.fsm

```diff
@@ -700,7 +701,7 @@ class FSM
   initState="init";
   finalStates[] =
   {
-    "End"
+    "End",
```

Comment:
@Windmolders I just copied the export out of the FSM editor once I made the changes... I presume the editor gives you the correct format/syntax.